Makefile

```
@@ -78,6 +78,8 @@ ALERTMANAGER            ?= $(GOBIN)/alertmanager-$(ALERTMANAGER_VERSION)
 MINIO_SERVER_VERSION    ?= RELEASE.2018-10-06T00-15-16Z
 MINIO_SERVER            ?=$(GOBIN)/minio-$(MINIO_SERVER_VERSION)
 
+IMPORTS_TO_AVOID				?=errors,prometheus/tsdb,prometheus/prometheus/pkg/testutils
```

I think it would be nice to have full modules path. E.g not everything is hosted in `github.com` domain.

Also: 
```suggestion
MODULES_TO_AVOID				?=errors,prometheus/tsdb,prometheus/prometheus/pkg/testutils
```